workbench-core/accounts/src/constants/projectStatus.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+// https://www.damirscorner.com/blog/posts/20200619-StringLiteralTypeGuardInTypescript.html
+// This file is structured this way so we can verify whether a user input is of type `ProjectStatus`
+export const PROJECT_STATUS: string[] = ['AVAILABLE', 'SUSPENDED', 'DELETED'];
+
+// Convert HOSTING_ACCOUNT_STATUS array to string literals
```

Comment:
`HOSTING_ACCOUNT_STATUS` => `PROJECT_STATUS`

Reply:
oop. thank you.